src/connectedk8s/azext_connectedk8s/_client_factory.py

```diff
@@ -22,9 +22,9 @@
 
 def cf_connectedk8s(cli_ctx, *_):
     from azext_connectedk8s.vendored_sdks import ConnectedKubernetesClient
-    if os.getenv('AZURE_ACCESS_TOKEN'):
+    if os.getenv(consts.Azure_Access_Token_Variable):
```

Comment:
Will changing from reading from environment variables to reading from configuration files have an impact on the user experience?
Do users have to manually fill in `Azure_Access_Token_Variable` in` _constants.py`?

Reply:
the value is still being read from environment variables - it is only the (hardcoded) environment variable name 'AZURE_ACCESS_TOKEN' that moved to constants as per Narayan's previous comments on this PR.
There is no change for users.